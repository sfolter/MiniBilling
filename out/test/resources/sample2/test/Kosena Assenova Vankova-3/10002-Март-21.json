{
    "documentDate": "2022-08-08T16:27:17.789475300",
    "documentNumber": "10002",
    "consumer": "Kosena Assenova Vankova",
    "reference": "3",
    "totalAmount": 3662.4,
    "lines": [
        {
            "index": 6,
            "quantity": 436,
            "lineStart": "2021-01-01T14:40:00Z",
            "lineEnd": "2021-03-11T07:00:00Z",
            "product": "gas",
            "price": 1.4,
            "priceList": 1,
            "amount": 610.4
        },
        {
            "index": 6,
            "quantity": 436,
            "lineStart": "2021-01-01T14:40:00Z",
            "lineEnd": "2021-03-11T07:00:00Z",
            "product": "gas",
            "price": 1.4,
            "priceList": 1,
            "amount": 610.4
        },
        {
            "index": 6,
            "quantity": 436,
            "lineStart": "2021-01-01T14:40:00Z",
            "lineEnd": "2021-03-11T07:00:00Z",
            "product": "gas",
            "price": 1.4,
            "priceList": 1,
            "amount": 610.4
        },
        {
            "index": 6,
            "quantity": 436,
            "lineStart": "2021-01-01T14:40:00Z",
            "lineEnd": "2021-03-11T07:00:00Z",
            "product": "gas",
            "price": 1.4,
            "priceList": 1,
            "amount": 610.4
        },
        {
            "index": 6,
            "quantity": 436,
            "lineStart": "2021-01-01T14:40:00Z",
            "lineEnd": "2021-03-11T07:00:00Z",
            "product": "gas",
            "price": 1.4,
            "priceList": 1,
            "amount": 610.4
        },
        {
            "index": 6,
            "quantity": 436,
            "lineStart": "2021-01-01T14:40:00Z",
            "lineEnd": "2021-03-11T07:00:00Z",
            "product": "gas",
            "price": 1.4,
            "priceList": 1,
            "amount": 610.4
        }
    ]
}