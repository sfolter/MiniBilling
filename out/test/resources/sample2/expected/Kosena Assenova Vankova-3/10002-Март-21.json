{
  "documentNumber": "10002",
  "consumer": "Kosena Assenova Vankova",
  "reference": "3",
  "totalAmount": 60.8,
  "lines": [
    {
      "index": 1,
      "quantity": 106.7,
      "lineStart": "2021-01-01T08:00:00Z",
      "lineEnd": "2021-02-28T21:59:59Z",
      "product": "gas",
      "price": 0.4,
      "priceList": 1,
      "amount": 42.68
    },{
      "index": 2,
      "quantity": 45.3,
      "lineStart": "2021-02-28T22:00:00Z",
      "lineEnd": "2021-03-25T13:40:00Z",
      "product": "gas",
      "price": 0.4,
      "priceList": 1,
      "amount": 18.12
    }
  ]
}